web-wallet/src/lib/stores/walletStore.js

```diff
@@ -129,24 +81,19 @@ async function setCurrentAddress(address) {
 
 /** @type {WalletStoreServices["stake"]} */
 const stake = async (amount, gasSettings) =>
-  syncedAction(() => {
-    // @ts-expect-error
-    return walletInstance.stake(
-      getCurrentAddress(),
-      amount,
-      new Gas(gasSettings)
-    );
-  });
+  asyncNoopFailure(amount, gasSettings);
 
 /** @type {WalletStoreServices["sync"]} */
-function sync(from) {
```

Comment:
Not implemented yet in w3sper?

Reply:
I don't get your comment, sorry.

Reply:
Was about line 87, but doesn't matter, as this should be resolved as part of the another PR.